lib/mongodb.js

```diff
@@ -2015,6 +2026,64 @@ MongoDB.prototype.ping = function(cb) {
   }
 };
 
+MongoDB.prototype.beginTransaction = function(isolationLevel, cb) {
+  // TODO: think about how to convert READ_COMMITED, etc. to transactionOptions
+  const transactionOptions = {
+    readPreference: 'primary',
+    readConcern: {level: 'local'},
+    writeConcern: {w: 'majority'},
+  };
+  if (typeof isolationLevel === 'object') {
```

Comment:
for nullSafety , you may use. Keep in mind that typeof returns true when checked against null
```ts
 if (isolationLevel instanceof Object) {
```

Reply:
old school. :) I'll change it ot instanceOf

Reply:
done